src/ensembl/src/shared/helpers/urlHelper.ts

```diff
@@ -0,0 +1,14 @@
+import windowService from 'src/services/window-service';
+
+// FIXME: this should be moved to a file with general functions
+export const ensureAbsoluteUrl = (url: string) => {
+  if (typeof window === 'undefined') {
+    // this helper only makes sense for client-side rendering
+    return url;
+  }
+  const location = windowService.getLocation();
+
+  return /^https?/.test(url)
```

Comment:
Few questions on this one:
1. Do we need the `s?` here? 
2. Are we making the protocol to be mandatory?
3. Would it be better to check if the path starts with `/` instead?

Reply:
Actually, I am being stupid.

The only reason I added this helper in this PR was to enable the `InlineSVG` component to request svg's from localhost (during development). Because in production I will be getting real data with absolute urls to svg's.

For some reason, I decided to convert relative urls to absolute urls here. But perhaps a better solution would have been to allow `apiService` to use relative urls as well.

What if instead of adding this helper I update apiService from [this](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/services/api-service.ts#L40):

```
const url = `${this.host}${endpoint}`;
```

to something like this:

```
const url = options.useUnmodifiedEndpoint ? endpoint : `${this.host}${endpoint}`;
```
?

We might still need a function that converts relative urls to absolute urls at some later point (e.g. when we need to add server-side rendering), but for now update to the apiService should be enough to do the job.

Reply:
How about using `options.preserveEndpoint` to keep it short?

Reply:
Updated.